Bridges/InvokableMiddleware.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace PHPPM\Bridges;
+
+use Psr\Http\Message\ServerRequestInterface;
+use RingCentral\Psr7;
+
+class InvokableMiddleware implements BridgeInterface
+{
+    use BootstrapTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function bootstrap($appBootstrap, $appenv, $debug)
+    {
+        $this->bootstrapApplicationEnvironment($appBootstrap, $appenv, $debug);
```

Comment:
Since it's more or less stateless, this could also be done in a separate class, maybe like this:

```
$environment = new ApplicationEnvironment($appBootstrap);
$this->middleware = $environment->bootstrap($appenv, $debug);
```

IMO, that's somewhat cleaner separation...

Reply:
@franzliedke akaiks the current version is equivalent to the old Psr7 bridge. Can we treat this a separate enhancement and merge the current state?